query-engine/connector-test-kit-rs/query-engine-tests/tests/writes/data_types/scalar_list/json.rs

```diff
@@ -16,7 +16,7 @@ mod json {
         schema.to_owned()
     }
 
-    #[connector_test(exclude(CockroachDb))]
+    #[connector_test(exclude(CockroachDb, Js))]
```

Comment:
Would be nice to add a comment as to why this is skipped for Js (this and other instances).

In TS functional test suite, comment is actually required field when skipping providers/runtimes and test will fail if it is not provided. Not sure if something like this will make sense for engines as well.

Reply:
I will open an issue to address each of those and soon remove the tags for those that are unnecessary and document the rest.